### ISSUE TITLE
[Feature Request] Improvements to Increase/Decrease Resolution Hotkeys

### ISSUE THREAD

**Issue by AsteroidPizza39** [Ryujinx/Ryujinx#5955]
### Overview

In the Keyboard Hotkeys section of Ryujinx's Settings, you can set a hotkey to increase or decrease the resolution. They work for what they say they do, but there are two features that feel like they're missing:

- Ability to set maximum and minimum resolution scales that can be attained through these hotkeys
- Ability to change the step count of the hotkey

### Smaller details

Currently these hotkeys increase or decrease your resolution by a factor of 1x each time. If your current resolution scale is 1x, decreasing it will move it to 4x, and increasing it will bring it up to 2x. When using a Custom resolution scale, such as 0.5x, using the hotkey to increase the resolution brings it to 1.5x, but the decrease hotkey will set the resolution to 4.5x. It isn't possible to use the hotkeys to set a resolution scale lower than 1. With this suggestion implemented, there could be a user-set limit that must be greater than 0x for the resolution, and the user could set their maximum based on what they want and what their hardware can reasonably handle. Being able to set a step count would allow for more precise changes in resolution, allowing you to go from 1 to 1.5 with one input, rather than being force to go from 1 to 2 or something similar.

### Nature of request

Feature Enhancement

### Why would this feature be useful?

These improvements would allow more options for increasing and decreasing the resolution. On systems with lower-end hardware, being able to limit the maximum resolution that can be switched to using the hotkeys would be especially helpful, since the short amount of time that a higher resolution is active could cause significant performance issues. Being able to set how much the hotkeys increase or decrease the resolution would also be helpful, making it easier to reach specific resolutions. I use the hotkeys so i can navigate to a location at a lower resolution, to increase performance, then upscale the resolution to take screenshots.
Simply put, it's a QOL improvement for users.
